WorkloadManagementSystem/DB/PilotAgentsDB.py

```diff
@@ -19,14 +19,15 @@
 
 __RCSID__ = "$Id$"
 
+from types import IntType, LongType, ListType
+import threading
```

Comment:
The lock does not seem to be used anywhere, is it ?

Reply:
See comments above: changes for this file are in the integration branch.